test/expect/TestOperators.test_concat2.expect

```diff
@@ -6,10 +6,10 @@ graph {
   node {
     input: "1"
     input: "2"
-    output: "3"
-    op_type: "Concat"
+    output: "4"
+    op_type: "cat"
```

Comment:
Also why now exports "cat", not "Concat"?

Reply:
Yeah, looks like we have lost the symbolic for cat somehow.